### PR TITLE
LPS-194066 [Marketplace] Adjusted Clarity page to be responsive

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf
ci:test:relevant
ci:test:stable

**Review by claricebrasil — APPROVED** [review]

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 1]
```diff
@@ -1,52 +1,248 @@
...
+		max-width: 400px;
+	}
+
+	.banner__contact-button {
+		width: 200px;
+	}
+
+	.banner__free-trial-button {
+		width: 200px;
+	}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:227]: Same here

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 2]
```diff
@@ -1,52 +1,248 @@
...
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:202]: Same

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 3]
```diff
@@ -1,52 +1,248 @@
...
+	white-space: nowrap;
 }
 
-.banner__buttons-container {
-	gap: 1.5rem;
+.liferay__icon {
+	display: none;
+	height: 32px;
+	width: 32px;
 }
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:70]:
```suggestion
.liferay__icon,
.share__icon 
{
	display: none;
	height: 32px;
	width: 32px;
}
```

**Review by joanagrego — APPROVED** [review]
I added some suggestions, but it's optional to follow them.
Thanks @welitonpe

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css [hunk 4]
```diff
@@ -14,6 +14,64 @@
...
+@media only screen and (min-width: 1200px) {
+	picture {
+		display: flex;
+		justify-content: center;
+	}
+
+	.hero-container__image-hero {
+		margin-top: 6rem;
+		width: 80%;
+	}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css:76]: Shouldn't we try to make it more generic? We're using the same style for all screen sizes. 

Try to use something like 
```suggestion
  @media only screen and (min-width: 600px) and (max-width: ?px)  {
    //Put your CSS here//
   
    }

```

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Review by joanagrego — APPROVED** [review]

**Comment by welitonpe** [comment]
ci:forward

**Comment by LuanaKSilva** [comment]
Hey @welitonpe, I have analyzed the errors pointed out in the relevant, none of them have to do with your PR. Feel free to push forward.

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html [hunk 5]
```diff
@@ -1,49 +1,65 @@
 [#assign
-	publisherEmail = configuration.publisherEmail
+publisherEmail = configuration.publisherEmail
```
Comment (jairmedeiros) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html:2]:
```suggestion
    publisherEmail = configuration.publisherEmail
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html [hunk 6]
```diff
@@ -1,49 +1,65 @@
...
 
-<div class="align-items-center banner d-flex justify-content-between p-4">
-	<div class="banner__content d-flex">
-		<img alt="App Logo" data-lfr-editable-id="image-editable-product-logo" data-lfr-editable-type="image" src="[resources:app_image.svg]" />
+<div class="banner p-3 rounded">
+	<div class="banner__content d-flex justify-content-between align-items-center">
+		<div class="d-flex flex-column justify-content-between w-100">
+			<div class="align-items-center d-flex w-100">
+				<div class="banner__content__app__image_container">
+				<img alt="App Logo" data-lfr-editable-id="image-editable-product-logo" data-lfr-editable-type="image" src="[resources:app_image.svg]" />
```
Comment (jairmedeiros) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.html:10]:
```suggestion
				     <img alt="App Logo" data-lfr-editable-id="image-editable-product-logo" data-lfr-editable-type="image" src="[resources:app_image.svg]" />
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
resent:[https://github.com/pt-liferay-solutions/liferay-portal/pull/4734](url)

**Pull request closed without merging**